azerothcore/azerothcore-wotlk
# fix(Core/Conditions): Add CONDITION_SOURCE_TYPE_QUEST_AVAILABLE

Cherry-picked from 
TrinityCore/TrinityCore@95f2aed
TrinityCore/TrinityCore@cef5d79
TrinityCore/TrinityCore@fe2db07
and adapted to AC


## CHANGES PROPOSED:
-  Merge CONDITION_SOURCE_TYPE_QUEST_ACCEPT & CONDITION_SOURCE_TYPE_QUEST_SHOW_MARK to CONDITION_SOURCE_TYPE_QUEST_AVAILABLE 


## ISSUES ADDRESSED:
- Ref https://github.com/azerothcore/azerothcore-wotlk/issues/1372



## TESTS PERFORMED:
- Build
- in-game



## HOW TO TEST THE CHANGES:

1. Log any char
2. .char lev 77
3. .go c id 30944
4. See that no quest marker (!) is available
5. .q add 13145
6. See that you now see a (?)
7. .q comp 13145
8. .q rew 13145
9. See that he now has 3 quests unlocked and (!) above his head

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
